packages/amplify_authenticator/lib/src/enums/confirm_signup_types.dart

```diff
@@ -0,0 +1,6 @@
+enum ConfirmSignUpType { username, password, email, phone_number, code }
+
+ConfirmSignUpType fromStringToConfirmSignUpType(String str) {
```

Comment:
Since `List.firstWhere` throws a `StateError` if it doesn't find the value, we probably want to either try/catch it or use the [firstWhereOrNull](https://pub.dev/documentation/collection/latest/collection/IterableExtension/firstWhereOrNull.html) function from `package:collection` then check if null in our widget or in the widget constructor, preferably.